### PR TITLE
Add initial home screen with hard coded data

### PR DESCRIPTION
**Pull request by DesislavaIT**
I've added the home screen PHP (HTML), CSS, and JS files. There are also two images added - `plus-icon.png` (needed) and `book.jpg` (for visibility, will be deleted later). In later MR the data will be synced with the database.

Screenshot 2024-05-25 at 1 59 17 PM
Screenshot 2024-05-25 at 1 59 29 PM
Screenshot 2024-05-25 at 1 59 36 PM
Screenshot 2024-05-25 at 1 59 46 PM
Screenshot 2024-05-25 at 1 59 53 PM
Screenshot 2024-05-25 at 2 00 10 PM

### CODE REVIEW

File: BorrowBooks/features/home/home.js [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+        item.addEventListener("click", function() {
+            const image = this.getAttribute("data-image");
+            const title = this.getAttribute("data-title");
+            const author = this.getAttribute("data-author");
+            const description = this.getAttribute("data-description");
+
+            console.log('Title:', title);
+            console.log('Author:', author);
+            console.log('Description:', description);
+            console.log('Image:', document.getElementById("bookImage").scr);
```
Comment (vasilkadt) [inline BorrowBooks/features/home/home.js]: This should be removed.

File: BorrowBooks/features/home/home.js [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+
+            document.getElementById("bookImage").scr = image;
+            console.log('Image:', document.getElementById("bookImage").scr);
```
Comment (vasilkadt) [inline BorrowBooks/features/home/home.js]: Should be removed.

File: BorrowBooks/features/home/home.js (hunk 1 above)
Comment (DesislavaIT) [reply BorrowBooks/features/home/home.js]: You are right. It was used for testing. The change is applied now.

File: BorrowBooks/features/home/home.js (hunk 2 above)
Comment (DesislavaIT) [reply BorrowBooks/features/home/home.js]: Done.

**Review by vasilkadt — APPROVED** [review]

**Pull request merged**